spec/index.adoc

```diff
@@ -4157,6 +4169,7 @@ ExecutionState = {
   wasm_state : WasmState;
   params : Params;
   response : NoResponse | Response;
+  controllers: List Principal;
```

Comment:
I wasn't sure if I need to somehow set this to the actual list of controllers or if this is fine given this is a model. Please let me know if I need to add something more.

Reply:
Fixed.